agir/front/components/authentication/Connexion/TellMore/TellMore.js

```diff
@@ -217,21 +222,19 @@ const TellMore = ({ dismiss }) => {
               />
             </div>
           </InputGroup>
-          <InputCheckbox onClick={toggleShowMandate}>
-            <input
-              type="checkbox"
+          <div style={{ marginTop: "0.625rem" }}>
+            <CheckboxField
               name="mandat"
-              checked={showMandate}
-              onChange={() => {}}
+              label="J'ai un mandat"
+              value={showMandat}
+              onChange={() => toggleShowMandat()}
```

Comment:
C'est mieux d'éviter le plus possible de déclarer des fonctions dans le render (sinon une nouvelle fonction est créée à chaque fois)
```suggestion
              onChange={toggleShowMandat}
```